terraform/tags.tf

```diff
@@ -0,0 +1,10 @@
+locals {
+  common_tags = module.tags.common_tags
+}
+
+module "tags" {
+  source      = "git::https://github.com/hmcts/terraform-module-common-tags.git?ref=master"
+  environment = "stg"
```

Comment:
should this be a var, incase we change the subscription? e.g. like testing in dev

Reply:
yeah good point. amended.